tools/tpm2_rsaencrypt.c

```diff
@@ -46,14 +47,13 @@ struct tpm_rsaencrypt_ctx {
     struct {
         UINT8 k : 1;
         UINT8 I : 1;
-        UINT8 o : 1;
         UINT8 c : 1;
-        UINT8 unused : 4;
+        UINT8 unused : 5;
```

Comment:
Is the unused field really needed here?

The bit-fields will be packed in an unsigned char anyways so I don't see the value in explicitly defining it (specially since you need to update it each time you add/remove a bit-field).